packages/xo-web/src/common/xo/copy-vm-modal/index.js

```diff
@@ -81,6 +92,7 @@ class CopyVmModalBody extends BaseComponent {
               <SelectCompression
                 disabled={copyMode !== 'fullCopy'}
                 onChange={this.linkState('compression')}
+                showZstd={isZstdSupported}
```

Comment:
I think `zstd` is clear. What do you think?

Reply:
`showZstd` is more significant than `zstd`.